There is inconsistency between delivered Makefile.in and Makefile.am
--- libconfig-1.6/lib/Makefile.am.1	2016-03-24 09:13:45.872081783 +0300
+++ libconfig-1.6/lib/Makefile.am	2016-03-24 09:13:57.022202777 +0300
@@ -18,7 +18,7 @@
 #
 # For more info see section 6.3 of the GNU Libtool Manual.
 
-VERINFO = -version-info 11:1:3
+VERINFO = -version-info 11:0:2
 
 ## Flex
 PARSER_PREFIX = libconfig_yy
